index.js

```diff
@@ -119,15 +130,31 @@ function sortPackageJson(packageJson, options = {}) {
     });
   }
 
-  function sortSubKey(key, sortList, unique) {
+  function sortSubKey({
+    key,
+    sort,
+    sortScripts,
+    unique
+  }) {
+    if (sort === false) {
+      return
+    }
+
     if (Array.isArray(packageJson[key])) {
       packageJson[key] = packageJson[key].sort();
       if (unique) {
         packageJson[key] = array_unique(packageJson[key]);
       }
       return;
     }
+
     if (typeof packageJson[key] === 'object') {
+      let sortList
+      if (sortScripts) {
+        sortList = compareScriptKeys
+      } else if (Array.isArray(sort)) {
+        sortList = sort
+      }
```

Comment:
This logic seems a little complicated. Can we just provide `sortList` as an option to the function?